contoso_supermarket/developer/freezer_monitoring/src/mqtt-simulator-deployment.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    io.kompose.service: mqtt-simulator
+  name: mqtt-simulator
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      io.kompose.service: mqtt-simulator
+  template:
+    metadata:
+      labels:
+        io.kompose.service: mqtt-simulator
+    spec:
+      containers:
+        - image: charrisagoracr.azurecr.io/mqtt-simulator:latest
```

Comment:
shouldn't this file be in the operations folder ?

Reply:
it is. this is an old version that I have now removed.